pkg/render/tenant.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package render
+
+import (
+	operatorv1 "github.com/tigera/operator/api/v1"
+	rmeta "github.com/tigera/operator/pkg/render/common/meta"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ Component = &tenantIsolater{}
+
+func NewTenantIsolater(c Component, ns string) Component {
```

Comment:
This func is not used anywhere?

Reply:
Oh yeah that's from an earlier prototype, will remove.